tests/integration/support/steps.go

```diff
@@ -0,0 +1,274 @@
+package support
+
+import (
+	"encoding/json"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/optimizely/go-sdk/optimizely/event"
+
+	"github.com/DATA-DOG/godog/gherkin"
+	"github.com/optimizely/go-sdk/optimizely/entities"
+	"github.com/optimizely/go-sdk/tests/integration/models"
+	"github.com/optimizely/go-sdk/tests/integration/optlyplugins"
+	"gopkg.in/yaml.v3"
+)
+
+// ScenarioCtx holds both apiOptions and apiResponse for a scenario
+type ScenarioCtx struct {
+	apiOptions  models.RequestParams
```

Comment:
I also meant for `models.RequestParams` to be refactored to `APIOptions` and same with the response params